src/styles/components/_menu.scss

```diff
@@ -1,10 +1,31 @@
-$interface-types: (
```

Comment:
This file has a lot of color things going on. let's see if we can change all of them to either use the `color` or `palette` mixin.